migrations/2_deploy_contracts.js

```diff
@@ -1,4 +1,4 @@
-var MetaCoin = artifacts.require("./BugBounty.sol");
+var BugBounty = artifacts.require("./BugBounty.sol");
```

Comment:
should stick to ES6 variable syntax `const` and `let` - this one would be `const`